pages/linux/fuser.md

```diff
@@ -6,3 +6,19 @@
 - Identify process using a TCP socket:
 
 `fuser -n tcp {{port}}`
+
+- Find which process accessing a directory:
+
+`fuser {{path_to_dir}}`
+
+- Kill and signal processes:
+
+`fuser -k {{path_to_dir}}`
+
+- Find which process is accessing a file:
+
+`fuser -m {{path_to_dir}}`
```

Comment:
The description mentions file, but the example shows directory. One or the other should be corrected.